packages/tools/src/tools/base/AnnotationTool.ts

```diff
@@ -310,6 +310,35 @@ abstract class AnnotationTool extends AnnotationDisplayTool {
     return typeof scalingModule?.suvbw === 'number';
   }
 
+  protected getAnnotationStyle(context: {
```

Comment:
Can you make all the annotation tools use this then? Length etc

Reply:
Updated angle, arrow, bidirectional, circle ROI, cobb angle, drag probe, elliptical ROI, key image, length, probe and rectangle tools.